src/bin/julialauncher.rs

```diff
@@ -134,12 +105,6 @@ fn get_julia_path_from_channel(
                 .installed_versions.get(version)
                 .ok_or_else(|| anyhow!("The juliaup configuration is in an inconsistent state, the channel {} is pointing to Julia version {}, which is not installed.", channel, version))?.path;
 
-            check_channel_uptodate(channel, version, versions_db).with_context(|| {
```

Comment:
Could you briefly explain, why you removed the online check from the `julialauncher` binary?

Reply:
Because If you use julialauncher offline or in bad/slow network condition. It would block the running or add a noticable delay. So removing online checking of julialauncher seems more reasonable. Make `juliaup` the explicit update manager. And `julialauncher` the simple launcher.